lib/snapshot/options.rb

```diff
@@ -44,17 +45,21 @@ def self.available_options
                                      end),
         FastlaneCore::ConfigItem.new(key: :languages,
                                      description: "A list of languages which should be used",
-                                     is_string: false,
-                                     default_value: [
-                                       'en-US'
-                                     ]),
+                                     short_option: "-g",
+                                     type: Array,
+                                     default_value: ['en-US'],
+                                     verify_block: proc do |value|
+                                       raise "'languages' must be an array" unless value.kind_of?(Array)
```

Comment:
Yes, I agree that it should be part of **fastlane_core** that if you specify `type: Array` that you get this check automatically.

However, since that code is not there yet, I just wanted to make these two parameters consistent with `:devices` so that at least all three make the same check.

When we refactor this into **fastlane_core**, we can remove it from here and all the other options. How does that sound?